xcode/build_phases/multiple_swiftlint/yaml_manager.rb

```diff
@@ -0,0 +1,24 @@
+require 'yaml'
+require 'fileutils'
+
+class YamlManager
+    def initialize(swiftlint_yaml_path)
+        @swiftlint_yaml_path = swiftlint_yaml_path
+        @configuration ||= YAML.load(File.read(@swiftlint_yaml_path))
+    end
+    
+    def get_configuration(key)
```

Comment:
use ruby subscript https://stackoverflow.com/questions/10018900/how-does-defining-square-bracket-method-in-ruby-work

Reply:
I don't get it.
this code is not work for me, work only like yamlManager.[](key), but I expected yamlManager.[key]

Reply:
🤔 ok